### DISCUSSION TITLE
How do I deactivate Hail as a decice admin?

### DISCUSSION THREAD

**Discussion by mrtootoo** [aistra0528/Hail#201] [Q&A] [answered]
![Screenshot_20240308-100449](https://github.com/aistra0528/Hail/assets/53195125/b0d7b484-23f6-4d12-808a-f3d27bef1609)

**Comment by aistra0528** [comment, marked as answer]
![RemoveDeviceOwner](https://github.com/aistra0528/Hail/assets/55940744/926654af-a06b-486d-bf6f-7de79dcd28c6)

**Reply by mrtootoo** [reply]
I tap button circled in red and get a box that display the word "Terminal".  What do I do from here?
![Screenshot_20240309-092852](https://github.com/aistra0528/Hail/assets/53195125/c4b60695-e451-47aa-8d42-144365fee325)

**Reply by aistra0528** [reply]
First change the working mode to "Device Owner" and tap the button.

**Comment by mrtootoo** [comment]
> First change the working mode to "Device Owner" and tap the button.



> First change the working mode to "Device Owner" and tap the button.

Thank you.  I did that.  I now see app is deactivated as a device admin and the Shizuku - Disable mode still works.  Thanks.
